std/selector/multiplexer2_test.go

```diff
@@ -0,0 +1,164 @@
+package selector
```

Comment:
See the suggested edits. I refactored a bit. Also I think we dont need the TestBench tests now when we collect statistics.

I'd also move the remaining tests here (for mux2/mux3 before renaming) to multiplexer_test.go file. And that test file we can change the package to be `selector` instead of `selector_test`